### ISSUE TITLE
persisting authentication error with github

### ISSUE THREAD

**Issue by FTeichmann** [runjak/TranscriptionDesk#13]
I still get invalid authentication errors after updating the database. Maybe its a redirect problem, maybe something worse. Will keep searching.

**Comment by FTeichmann** [comment]
Update: fixed by furious clicking on the login-button. As this all-time recommended strategy is not the developers friend, we should keep looking for the bug.

**Comment by runjak** [comment]
Was there anything in your log files about this?
Try something like ```cd /var/log/apache2; tail -f access_log error_log```, and see if reproducing the error yields anything interesting pls. Or lets see if we can reproduce this together when we next meet.

**Comment by FTeichmann** [comment]
Log file states:
```[Sun Jul 19 09:59:13.929765 2015] [:error] [pid 13943] [client 127.0.0.1:42256] PHP Warning:  include_once(../../navbar.php): failed to open stream: No such file or directory in /home/franz/workspace/TranscriptionDesk/src/auth/callback/authenticationError.php on line 7, referer: http://localhost/workspace/TranscriptionDesk/src/
[Sun Jul 19 09:59:13.967225 2015] [:error] [pid 13943] [client 127.0.0.1:42256] PHP Warning:  include_once(): Failed opening '../../navbar.php' for inclusion (include_path='.:/usr/share/php:/usr/share/pear') in /home/franz/workspace/TranscriptionDesk/src/auth/callback/authenticationError.php on line 7, referer: http://localhost/workspace/TranscriptionDesk/src/```
Error seems to happen right after reactivating browser from sleep mode before refreshing the github tab. I assume it's the outdated token our page tries to access.

**Comment by runjak** [comment]
Yes, that sounds possible. After authentication however, TranscriptionDesk starts its own session with a client and thus doesn't need to check GitHub or any other authentication provider any further after login. This implies that this issue would be confined to a situation where a GitHub token would be outdated and our site would be accessed before GitHub, if I understand correctly.

**Comment by RPOD** [comment]
Had the same problem happen to me currently too. Try to revoke the transcriptiondesk application from github and log in after that.
Also make sure you use ```127.0.0.1:8080``` instead of ```localhost:8080``` which helped me the other day.
